spec/poros/food_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+  it "exists", :vcr do
+    servce = FoodsService.food_search("sweet potatoes")
```

Comment:
No need to make this service call -- by calling the facade, it is going to call the service
However, to make this a pure unit test, it's best practice to create the test data that you pass into the PORO by copying the JSON response sent back from the API and creating a hash here or in a fixture. Checkout the PORO tests in Congress tracker as a reference